docs/CONFIG-VARS.md

```diff
@@ -351,7 +351,7 @@ Additional documentation for the SAS Workload Orchestrator Service can be found
 | Name | Description | Type | Default | Required | Notes | Tasks |
 | :--- | ---: | ---: | ---: | ---: | ---: | ---: |
 | V4_CFG_CLUSTER_NODE_POOL_MODE | The mode of cluster node pool to use | string | "standard" | false | [standard, minimal] | viya |
-| V4_CFG_EMBEDDED_LDAP_ENABLE | Deploy OpenLDAP in the namespace for authentication | bool | false | false | [Openldap Config](../roles/vdm/templates/generators/openldap-bootstrap-config.yaml) | viya |
+| V4_CFG_EMBEDDED_LDAP_ENABLE | Deploy OpenLDAP in the namespace for authentication | bool | false | false | [Openldap Config](../roles/vdm/templates/generators/openldap-bootstrap-config.yaml). If you set this value to false, you must set `V4_CFG_SITEDEFAULT` to point to a sitedefault file which contains values applicable for your authentication configuration. | viya |
```

Comment:
Since the default for `V4_CFG_EMBEDDED_LDAP_ENABLE` is false, should we alter the wording to say it a bit differently, like "If you do not set this value to true, you must set `V4_CFG_SITEDEFAULT` to point to a ..."

Reply:
I agree, that makes sense since the default value will be false, so a user may not explictly "set" this value. 

Updated the wording